cmd/bnsd/scenarios/bnsdtest/bnsdtest.go

```diff
@@ -22,114 +25,150 @@ import (
 	"github.com/iov-one/weave/x/escrow"
 	"github.com/iov-one/weave/x/multisig"
 	"github.com/stellar/go/exp/crypto/derivation"
-	abci "github.com/tendermint/tendermint/abci/types"
-	cfg "github.com/tendermint/tendermint/config"
 	"github.com/tendermint/tendermint/libs/log"
 	nm "github.com/tendermint/tendermint/node"
-	"github.com/tendermint/tendermint/rpc/test"
 	tm "github.com/tendermint/tendermint/types"
 )
 
-// application version, will be set during compilation time
+// EnvConf is a work in progress collection of previously global variables.
+// This is to be cleaned up in the following updates.
+type EnvConf struct {
+	Alice       *client.PrivateKey
+	ChainID     string
+	AntiSpamFee coin.Coin
```

Comment:
All `EnvConf` attributes are a copy/paste of the previous global state. No change there, although I question some of them myself.
No way to change any of the _right now_, because global variables were not possible to change as well. I am planning to add function arguments to the `StartBnsd` function so that the configuration can be changed and extended without breaking existing code.